lib/l10n/app_en.arb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I believe this app is only in French. If so please have only French file.

Reply:
Done.